music/urls.py

```diff
@@ -2,18 +2,9 @@
 from . import views
```

Comment:
No need to remove all these existing urls, views; create an `api` directory in the apps and add urls, views and serializers files to it